web/template/mlpa-documents.gohtml

```diff
@@ -87,7 +87,7 @@
                         <td class="govuk-table__cell">
                             {{ if eq .Direction "Incoming" }}
                                 {{ .Type }}
-                                {{ if .SubType }}<br />&ndash; {{ .SubType }}{{ end }}
+                                {{ if .SubType }}<br />&ndash; {{ subtypeShortFormat .SubType }}{{ end }}
```

Comment:
This is a document subtype, which is already a human-readable string (e.g. "Reduced fee request evidence") so doesn't need to be translated
```suggestion
                                {{ if .SubType }}<br />&ndash; {{ .SubType }}{{ end }}
```

Reply:
I feel like this should have some sort of test around it.